src/App.tsx

```diff
@@ -1,24 +1,291 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import {
+  FC,
+  ChangeEvent,
+  FormEvent,
+  useEffect,
+  useRef,
+  useState,
+  useMemo,
+  useCallback,
+} from 'react';
+import cn from 'classnames';
+import './App.scss';
 import { UserWarning } from './UserWarning';
+import {
+  getTodos,
+  removeTodo,
+  addTodoToServer,
+  changeTodo,
+} from './api/todos';
+import { TodosList } from './components/TodosList/TodosList';
+import { Todo } from './types/Todo';
+import { ErrorInfo } from './components/ErrorInfo/ErrorInfo';
+import {
+  preparedTodos,
+  getcompletedTodosIds,
+  filterTodosByCompletion,
+} from './utils/todoUtils';
+import { StatusValue } from './types/StatusValue';
 
-const USER_ID = 0;
+const USER_ID = 10725;
+
+export const App: FC = () => {
+  const formRef = useRef<HTMLInputElement | null>(null);
+
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [queryTodo, setQueryTodo] = useState('');
```

Comment:
Should it be here?

Reply:
@silchyn what do you mean? All these states we need in that component